python/mp/rosary.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/python3
+import code
+import copy
+import threading
+import time
+import math
+
+from mp import color
+from .effects import *
+from pythonosc import udp_client
+from pythonosc import osc_bundle_builder
+from pythonosc import osc_message_builder
+
+class Bead:
+    def __init__(self, index=0):
+        self.index = index
+        self.color = color.Color()
+        
+    def __repr__(self):
+        return "Bead(index={}, color={})".format(self.index, self.color)
+
+    def copy_color(self, color):
+        self.color = copy.copy(color)
```

Comment:
I debated whether the Bead ought to be in its own class, but since nothing inherits from it (and probably nothing ever will), I think I agree that we should leave it as-is.